src/robot_dart/graphics/camera_osr.hpp

```diff
@@ -46,35 +49,22 @@ namespace robot_dart {
         class CameraOSR : public BaseGraphics {
 
         public:
-            CameraOSR(const dart::simulation::WorldPtr& world, unsigned int width = 640, unsigned int height = 480, bool shadowed = true) : _world(world), _width(width), _height(height), _frame_counter(0), _enabled(true), _buffer_valid(true), _recording(false), _filename("camera_record.png"), _image(new osg::Image)
+            CameraOSR(const dart::simulation::WorldPtr& world, unsigned int width = 640, unsigned int height = 480, bool shadowed = true) : _world(world), _width(width), _height(height), _frame_counter(0), _enabled(true), _buffer_valid(true), _recording(false), _filename("camera_record.png"), _image(new osg::Image), _shot_done(false)
 
             {
+                if (!global::pbufferManager) {
+                    std::cout << "Error: pbufferManager seems not to be started" << std::endl;
+                    assert(0);
```

Comment:
Maybe you can use the robot_dart assertions for this (and the rest assert(0))..

Reply:
ok